GameDesign/Assets/Scripts/BossAnger.cs

```diff
@@ -0,0 +1,82 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BossAnger : MonoBehaviour
+{
+    private static int MAX_NUMBER_ANGRY = 10; //To be defined on each level
+    private static int MAX_NUMBER_GAME_OVER = 6; //To be defined on each level
+
+    public static int angryCounter;
```

Comment:
If this class has static variables, how are we going to set them up for each level? Maybe is better to have them non-static

Reply:
I believe that if we add the [SerializeField], it doesn't have to be public, but they will need to be not static, otherwise they will change at class level, not at object level